### PR TITLE
Update for 3.19

### PR DESCRIPTION
**Pull request by 0xE0D59**
Added safeguards against null references. Reordered logic to be more efficient.

### CODE REVIEW

File: Core.cs [hunk 1]
```diff
@@ -38,44 +51,51 @@ public override void Render()
...
+                        throw new Exception("HasComponent<Buffs>() == true, but GetComponent<Buffs>() == null");
+                    var buffsList = buffs.BuffsList;
+                    if (buffsList.Count > 0 && IgnoreBuffs.Any(b1 =>
+                        buffsList.Any(b2 => string.Equals(b1, b2.Name, StringComparison.OrdinalIgnoreCase)))) return false;
+                }
 
+                if (!entity.HasComponent<Actor>()) return false;
+                var actor = entity.GetComponent<Actor>();
+                if (!FilterNullAction(actor)) return false;
+                if (actor.CurrentAction?.Skill?.Name == "AtziriSUmmonDemons") return false;
```
Comment (TehCheat) [inline Core.cs]: Is that U in `AtziriSUmmonDemons` supposed to be capitalized?

Comment (TehCheat) [reply Core.cs]: Looks like in the old line 62 it wasn't, so I'm guessing just a typo.

**Comment by 0xE0D59** [comment]
Fixed.

File: Core.cs (hunk 1 above)
Comment (0xE0D59) [reply Core.cs]: Fixed.

**Pull request merged**